dirsrvtests/tests/suites/filter/filter_test.py

```diff
@@ -422,7 +425,7 @@ def test_match_large_valueset(topology_st):
     etime = float(search_result[1].split('etime=')[1])
     log.info("Duration of the search from access log was %f", etime)
     assert len(entries) == groups_number
-    assert (etime < 1)
+    assert (etime < 2)
```

Comment:
When the server does not use the sorted valueset the etime is much higher (several tens of seconds) so may be we can put 5 seconds in case the CI box is very slow.

Reply:
Thanks, changed it to 5.